test/unit/base/module/LinkDialog.spec.js

```diff
@@ -49,7 +49,7 @@ describe('LinkDialog', () => {
       dialog.show();
 
       var checked = dialog.$dialog.find('#sn-checkbox-open-in-new-window').is(':checked');
-      expect(checked).to.be.false;
+      expect(checked).to.be.true;
```

Comment:
Is this an intended change?

Reply:
It does not seem to be related to the commit. strange.

Reply:
I modified above code to use createRange in LinkDialog, but I always get an error